From ehovland@s383.jpl.nasa.gov Thu Oct  6 10:49:43 2005
Date: Thu, 6 Oct 2005 10:45:30 -0700
From: Erik Hovland <erik@hovland.org>
To: gregkh@suse.de
Cc: trivial@rustcorp.com.au
Subject: [trivial patch] kobject_uevent.c has a typo in a comment
Message-ID: <20051006174530.GG25998@mage.jpl.nasa.gov>

This patch changes trough to through in a comment in kobject_uevent.c.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 lib/kobject_uevent.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/lib/kobject_uevent.c
+++ gregkh-2.6/lib/kobject_uevent.c
@@ -54,7 +54,7 @@ static char *action_to_string(enum kobje
 static struct sock *uevent_sock;
 
 /**
- * send_uevent - notify userspace by sending event trough netlink socket
+ * send_uevent - notify userspace by sending event through netlink socket
  *
  * @signal: signal name
  * @obj: object path (kobject)
